ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/classload/visitors/TypeSignatureTransformer.java

```diff
@@ -107,7 +108,7 @@ protected void visitField(ObjectFieldSymbol fieldSymbol) {
         StringJoiner joiner = new StringJoiner(" ");
         fieldSymbol.qualifiers().forEach(qualifier -> joiner.add(qualifier.getValue()));
         String signature = joiner.add(transformType(fieldSymbol.typeDescriptor()))
-                .add(fieldSymbol.name()).toString();
+                .add(fieldSymbol.getName().get()).toString();
```

Comment:
Is it confirmed at this point, that we are getting the field name? If so let's add a comment or let's use `orElseThrow`

Reply:
Yes, since it's a field symbol, we know that it always has a name.